File: Sem4/MPP_Templates/ZboruriCSharp/networking/ZborDTO3.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace networking
{
    [Serializable]
    public class ZborDTO3
    {
        public string destinatie{ get; set; }
        public string data { get; set; }
        public string ora { get; set; }

        public ZborDTO3(string destinatie, string data, string ora)
        {
            this.destinatie = destinatie;
            this.data = data;
            this.ora = ora;
        }
    }
}

File: Sem4/MPP_Templates/Zboruri/FlightNetworking/src/main/java/dto/ZborDTO3.java
package dto;

import java.io.Serializable;

public class ZborDTO3 implements Serializable {
    private String destinatie;
    private String data;
    private String ora;

    public ZborDTO3(String destinatie, String data, String ora) {
        this.destinatie = destinatie;
        this.data = data;
        this.ora = ora;
    }

    public String getDestinatie() {
        return destinatie;
    }

    public String getData() {
        return data;
    }

    public String getOra() {
        return ora;
    }
}
